packages/runtime-dom/src/apiCustomElement.ts

```
@@ -320,22 +325,47 @@ export class VueElement
     }
   }
 
+  private _unmount(keepParentAndOb?: boolean) {
+    if (!keepParentAndOb) {
+      this._parent = undefined
+      if (this._ob) {
+        this._ob.disconnect()
+        this._ob = null
+      }
+    }
+    this._app && this._app.unmount()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)